tests/gold_tests/autest-site/trafficserver.test.ext

```diff
@@ -221,6 +221,11 @@ def MakeATSProcess(obj, name, command='traffic_server', select_ports=True, enabl
     tmpname = os.path.join(config_dir, fname)
     p.Disk.File(tmpname, id=make_id(fname), typename="ats:config")
 
+    # for describe-float, used by conf_remap plugin
+    fname = "mysterio.config"
```

Comment:
Can't use that name, it's internal. Use something like "delain.config". "Delain" is *always* a good choice for names in examples.

Reply:
Changed to relain.